packages/react/src/auto/polaris/inputs/PolarisAutoJsonInput.tsx

```diff
@@ -0,0 +1,31 @@
+import { TextField } from "@shopify/polaris";
+import type { CSSProperties } from "react";
+import React from "react";
+import type { Control } from "react-hook-form";
+import { useFocus } from "../../../useFocus.js";
+import { useJSONInputController } from "../../hooks/useJSONInputController.js";
+
+export const PolarisAutoJSONInput = (props: {
+  field: string; // The field API identifier
+  control?: Control<any>;
+  style?: CSSProperties;
+}) => {
```

Comment:
Same here as the MUI variant. Could you add `& Partial<TextFieldProps>` to the props type and pass it in?